### PR TITLE
update node model to include description and cost code with create and update actions

### PR DESCRIPTION
**Pull request by harrietc52**

### CODE REVIEW

File: app/controllers/nodes_controller.rb [hunk 1]
```diff
@@ -6,13 +6,17 @@ def show
...
 		p = create_params
-		node = Node.create!(p)
-		redirect_to node_path(node.parent_id)
+		@node = Node.new(p)
+		if @node.save
+			flash[:success] = "Node created"
+		else
+			flash[:danger] = "Failed to create node"
+		end
+		redirect_to node_path(@node.parent_id)
```
Comment (emrojo) [inline app/controllers/nodes_controller.rb]: Update() has a different way of redirect_to depending on the possible problems while saving the node.  It's better if you keep the same convention (either redirect with error or always redirect to the parent like here)

File: app/views/nodes/show.html.erb [hunk 2]
```diff
@@ -9,6 +9,12 @@
   <h2><%= @node.name %></h2>
 <% end %>
 
+<% @node.attributes.slice("description", "cost_code").map do |attribute, name| %>
+  <% if name.present? %>
+    <p><b><%= attribute.gsub('_', ' ').capitalize %>:</b> <%= name %></p>
+  <% end %>
+<% end %>
+
```
Comment (emrojo) [inline app/views/nodes/show.html.erb]: Better use the internationalization variables to get the text for description and cost_code. Take a look at http://guides.rubyonrails.org/i18n.html

**Comment by jbeast** [comment]
Are there still changes that need to be done to this?

**Comment by jbeast** [comment]
Is it possible to filter where cost code is null through the API?

**Pull request merged**